src/get-started/java-binding.md

```diff
@@ -781,6 +798,10 @@ To test API implemented with Vertx tools, use the tools described in the
 [project documentation](https://vertx.io/docs/vertx-junit5/java).
 You can use [Vertx Web Client][vertx-web-client] as a client or a different
 HTTP client.
+API can be tested with either unit tests with mocks of services (no TestKit
+required), or with integration tests that use TestKit. For that, TestKit
```

Comment:
@Maria-Nosyk could you help with a better wording please?

Reply:
see my comment above